iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/xcshareddata/xcschemes/Regression Tests.xcscheme

```diff
@@ -14,8 +14,7 @@
       shouldAutocreateTestPlan = "YES">
       <Testables>
          <TestableReference
-            skipped = "NO"
-            parallelizable = "YES">
+            skipped = "NO">
```

Comment:
Was this causing multiple simulators to fire up ?. If we disable it the test might take too long to run. Is that what we want ?

Reply:
just testing if that was one of the causes of failure since Bitrise was failing as well